test/algorithms/hard-filter-min-platform-feature.test.js

```diff
@@ -61,6 +61,17 @@ var infraContainersNfsAutomountTestServers = common.genServers([
 	['b4343a95-5aeb-499a-bc0e-701c5119b89e', '7.0', '20170925T211846Z']
 ]);
 
+var infraContainersVolapiNfsV2TestServers = common.genServers([
```

Comment:
Should fix (c) notice and add 2020 as the year, now that we're here.

Btw, how in hell can I add a comment to a not modified line here?